core/src/main/java/org/mapfish/print/servlet/job/PrintJob.java

```diff
@@ -127,7 +128,11 @@ protected abstract PrintJobResult createResult(
   @Override
   public final PrintJobResult call() throws Exception {
     SecurityContextHolder.setContext(this.securityContext);
-    final Timer.Context timer = this.metricRegistry.timer(getClass().getName() + ".call").time();
+    final Timer.Context timer =
+        this.metricRegistry
+            .timer(
+                MetricRegistry.name(getClass().getSimpleName(), "ActualReportGenerationDuration"))
```

Comment:
I don't know what actual refer to?
```suggestion
                MetricRegistry.name(getClass().getSimpleName(), "ReportGenerationDuration"))
```

Reply:
It's not I understand by reading the metric name, what about `OnlyReportGenerationDuration`?

Reply:
I will go with real instead of actual to avoid french ambiguity